cmd/helm/list.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var listHelp = `
-This command lists all of the releases.
+This command lists all of the releases in specified namespace (uses current namespace context otherwise).
```

Comment:
"This command lists all of the releases in specified namespace (uses current namespace context otherwise)." --> "This command lists all of the releases for a specified namespace (uses current namespace context if namespace not specified)."